app/controllers/bars_controller.rb

```diff
@@ -18,33 +18,33 @@ def downvote
   def index
     @bars = Bar.page(params[:page])
     n = 0
-    # @bars.each do |bar|
-    #   @bar_name = bar.name
-    #   @bar_name += " #{bar.location}"
-    #   @bar_name.gsub(" ","+")
-    #   url = "http://www.google.com/search?q="+"#{@bar_name}"+"&num=10"
-    #   agent = Mechanize.new { |agent| agent.user_agent_alias = "Mac Safari" }
-    #   html = agent.get(url).body
-    #   doc = Nokogiri::HTML(html)
-    #   doc_string = doc.to_s
-    #   doc_find = nil
-    #   if result = doc_string.match(/lubh-bar( _...)/)
-    #     doc_find = result.captures
-    #     div_end = doc_find[0]
-    #     div_find = "lubh-bar"+"#{div_end}"
-    #     doc_at = doc.xpath("//div[@class=\"#{div_find}\"]")
-    #   end
-    #   if doc_at.nil?
+    @bars.each do |bar|
+      @bar_name = bar.name
+      @bar_name += " #{bar.location}"
+      @bar_name.gsub(" ","+")
+      url = "http://www.google.com/search?q="+"#{@bar_name}"+"&num=10"
```

Comment:
Surrounding space missing for operator `+`.<br>Prefer `to_s` over string interpolation.